src/server/protocol.ts

```diff
@@ -2700,6 +2701,42 @@ export interface InlayHintsResponse extends Response {
     body?: InlayHintItem[];
 }
 
+export interface MapCodeRequestArgs {
+    /// The files and changes to try and apply/map.

```

Comment:
Ehhh why the 3 slashes? It seems like documentation in this file uses JSDoc-y comments `/** */`.